packages/tools/client-debug-view/src/Debugger.tsx

```diff
@@ -88,12 +88,13 @@ export function FluidClientDebugger(props: FluidClientDebuggerProps): React.Reac
 	return (
 		<Resizable
 			style={{
-				position: "fixed",
+				position: "absolute",
```

Comment:
Just curious: what were these changes needed for?

Reply:
Set position as `absolute` will allow view, like "container dropdown list", to place in front of debugger background view. The render order will base on `z-index`.

Reply:
Ooh, gotcha. Sounds good!